docs/cloud/manage/plans.md

```diff
@@ -13,6 +13,16 @@ At Netdata, we believe in providing free and unrestricted access to high-quality
 
 We also provide paid subscriptions that designed to provide additional features and capabilities for businesses that need tighter and customizable integration of the free monitoring solution to their processes. These are divided into three different plans: **Pro**, **Business**, and **Enterprise**. Each plan will offers a different set of features and capabilities to meet the needs of businesses of different sizes and with different monitoring requirements.
 
+:::note
+To not disrupt current users space's access rights we will be keeping their current plan as the **Early Bird** plan. The reason for this is to allow users to 
+keep using the legacy **Member** role with the exact same permissions as it has currently.
+
+When you moving to another plan starting from the **Early Bird** plan users won't be able to return to it. The **Community** free plan will be there but this doesn't allow
+you to invite or change users using the Member role. See more details on our [roles and plans](https://github.com/netdata/netdata/blob/master/docs/cloud/manage/role-based-access-model.md#what-roles-are-available) documentation.
+
+**Early Bird** is a plan with more advantages than the **Community**, to avoid possible issues based on the above we currently don't users from this plan to go to the **Community**.
```

Comment:
```suggestion
```
It would be better to delete this, as we already said that the deprecated `Member` role is available only on the `Early Bird` plan. There is no reason to say that we are bringing disadvantages to users, they should be able to figure our this by themself.

Reply:
yes, will remove that and add a missing "allow" that I just noticed